cbdr/cortex
# COR-674: Decouple Labels from Fields

**Purpose:**
Decouple Labels from Fields in Cortex Forms so that the label isn't hidden when clicked into. Additionally add tooltips with helpful information for the User, based on the Wizard Decorator, to the form.

> As a user, I don't need the field title to appear in the field, I actually need it to live above the field so that I can always know what it is I'm filling out and if it's empty, I know what to put in it.

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/COR-674

**Changes:**
* Changes to setup
  * Use the linked Cortex-Plugins-Core branch to get the full result

* Architectural changes
  * Updates to Cortex-Plugins-Core

* Migrations
  * N/A
  
* Library changes
  * Cortex-Plugins-Core: https://github.com/cortex-cms/cortex-plugins-core/pull/53

* Side effects
  * N/A

**Screenshots**
* Before
<img width="1133" alt="screen shot 2017-04-05 at 9 59 33 am" src="https://cloud.githubusercontent.com/assets/8419757/24711794/9a088fda-19e6-11e7-9863-5f7c1dad3351.png">

* After
<img width="727" alt="screen shot 2017-04-05 at 9 58 39 am" src="https://cloud.githubusercontent.com/assets/8419757/24711757/7d8399c2-19e6-11e7-9e5d-4cb160cdabe7.png">

**QA Links:**
Gimme a second on this one, C66 is down at the moment...

**How to Verify These Changes**
* Specific pages to visit
  * Create a new Employer Blog post

* Steps to take
  * Click on various text fields to see the floating labels happen
  * Hover over tooltips to see the tooltip message

* Responsive considerations
  * N/A

**Relevant PRs/Dependencies:**
Cortex Plugins Core: https://github.com/cortex-cms/cortex-plugins-core/pull/53

**Additional Information**
N/A
